namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-dev/resources/rds.tf

```diff
@@ -37,3 +37,44 @@ resource "kubernetes_secret" "rds-instance" {
   }
 }
 
+module "rds-read-replica" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.6"
+
+  cluster_name         = var.cluster_name
+  cluster_state_bucket = var.cluster_state_bucket
+
+  application            = var.application
+  environment-name       = var.environment-name
+  is-production          = var.is-production
+  infrastructure-support = var.infrastructure-support
+  team_name              = var.team_name
+
+  replicate_source_db = module.rds-instance.db_identifier
+
+  providers = {
+    # Can be either "aws.london" or "aws.ireland"
+    aws = aws.london
+  }
+
+  db_parameter = [
+    {
+      name         = "rds.force_ssl"
+      value        = "0"
+      apply_method = "immediate"
+    }
+  ]
+}
+
+
+resource "kubernetes_secret" "rds-read-replica" {
+  metadata {
+    name      = "read-rds-instance-hmpps-book-secure-move-api-dev"
+    namespace = var.namespace
+  }
+
+  data = {
+    access_key_id     = module.rds-instance.access_key_id
+    secret_access_key = module.rds-instance.secret_access_key
+    url               = "postgres://${module.rds-instance.database_username}:${module.rds-instance.database_password}@${module.rds-read-replica.rds_instance_endpoint}/${module.rds-read-replica.database_name}"
```

Comment:
Note to reviewer: I'm using  the `database_username` and `database_password` from the active database with the endpoint for the replica, so that I can create a ready to use db_url for the replica, as the read-replica does not  expose a password, for obvious reasons.